File: print-manager-dotnet-core-cs/SecuredExample.cs

using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    internal class SecuredExample
    {
        public static void Run()
        {
            InputPdf inputPdf = new InputPdf(Util.GetPath("Resources/secured.pdf"), "password");
            PrintJob printJob = new PrintJob(Printer.Default, inputPdf);
            printJob.Print();
        }       
    }
}

File: print-manager-dotnet-core-vb/SecuredExample.vb
Imports ceTe.DynamicPDF.Printing

Friend Class SecuredExample
        Public Shared Sub Run()
            Dim inputPdf As New InputPdf(Util.GetPath("Resources/secured.pdf"), "password")
            Dim printJob As New PrintJob(Printer.Default, inputPdf)

            printJob.Print()
        End Sub
End Class